padl/transforms.py

```diff
@@ -508,6 +514,32 @@ def f(x):
 
         return graph
 
+    def _pd_process_traceback(self):
+        """Find where the Transform was defined (file, lineno, file) given the traceback. """
+        a_tb = None
+        for a_tb in self._pd_traceback[::-1]:
+            if 'padl' in a_tb[0]:
+                continue
+            break
+        return f'{a_tb.filename} in {a_tb.name}\n----> {make_green(a_tb.lineno)}    {a_tb.line}'
+
+    def _pd_get_error_idx(self):
+        """Get what element of a {class}`Transform` is failing if an Exception is produced during
+        an execution.
+
+        Subclasses of {class}`padl.transforms.Transform` need to implement this method.
```

Comment:
Thanks for the docstring! I think you accidentally used the wrong formatting for `{class}` in two places. I think it should be `:class:`.

Reply:
good catch